invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/ItemDetails/state/actions.js

```diff
@@ -1,14 +1,27 @@
-import { IS_LOADING, SUCCESS, HAS_ERROR } from './types';
-import { item } from '../../../../common/api';
-import { sendErrorNotification } from '../../../../common/components/Notifications';
+import { ES_DELAY } from '../../../../common/config';
+import {
+  IS_LOADING,
+  SUCCESS,
+  HAS_ERROR,
+  DELETE_IS_LOADING,
+  DELETE_SUCCESS,
+  DELETE_HAS_ERROR,
+} from './types';
+import { item as itemApi } from '../../../../common/api';
+import { BackOfficeURLS } from '../../../../common/urls';
+import {
+  sendErrorNotification,
+  sendSuccessNotification,
+} from '../../../../common/components/Notifications';
+import history from '../../../../history';
 
 export const fetchItemDetails = itemPid => {
   return async dispatch => {
     dispatch({
       type: IS_LOADING,
     });
 
-    await item
+    await itemApi
```

Comment:
We need to refactor that using try/catch and await no?

Reply:
@zzacharo I was tempted to do these changes also since I changed the underlying api call, but the cool thing about it is that async still returns a promise so nothing breaks. I think it will be more preferable to change them all (including the tests) in a dedicated issue.

Reply:
@ntarocco was saying to refactor every time we introduce new code just not to have a huge change in the end(plus creating an issue ofc to refactor the existing codebase) that's why I mentioned it ;)